e2e/bitrise.yml

```diff
@@ -0,0 +1,20 @@
+format_version: "11"
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+workflows:
+  test_get_identity_token:
+    steps:
+    - path::./:
+        inputs:
+        - subject: A
+        - audience: B
+    - script:
```

Comment:
NIT: check-output internal step could be used here: https://github.com/bitrise-steplib/bitrise-step-check-step-outputs/blob/main/step.yml#L18